packages/joint-core/src/dia/Graph.mjs

```diff
@@ -396,6 +396,41 @@ export const Graph = Model.extend({
         this.get('cells').remove(cell, { silent: true });
     },
 
+    transferCellEmbeds: function(sourceCell, targetCell, opt = {}) {
+
+        const batchName = 'transfer-embeds';
+        this.startBatch(batchName);
+
+        opt.reparent = true;
+
+        // Embed children of the source cell in the target cell.
+        const children = sourceCell.getEmbeddedCells(opt);
```

Comment:
Why do we pass `opt` here? Do we want `deep: true` to be use here?
That would work with `{ deep: true, reparent: true }` but I am not sure if this is a scenario we want to support.